charts/sonarqube-dce/README.md

```diff
@@ -117,6 +117,11 @@ The following table lists the configurable parameters of the Sonarqube chart and
 | `searchNodes.image.pullPolicy` | search image pull policy | `IfNotPresent` |
 | `searchNodes.image.pullSecret` | search imagePullSecret to use for private repository | `nil` |
 | `searchNodes.env` | Environment variables to attach to the search pods | `nil` |
+| `searchNodes.searchAuthentication.enabled` | Securing the Search Cluster with basic authentication and TLS in between search nodes | `false` |
+| `searchNodes.searchAuthentication.keyStoreSecret` | Existing Keystore/Truststore to be used | `""` |
+| `searchNodes.searchAuthentication.keyStorePassword` | Password to said Keystore/Truststore if set (optional) | `""` |
```

Comment:
Why not just simply put : `Password to Keystore/Truststore used in search nodes (optional)` also it would be worth to mention what type of format we expect : `pkcs12`

This properties assumes that KeyStore and Truststore password is the same, though it could be different files with different password, is that intended?

Reply:
As long as we don't support `xpack.security.transport.ssl.verification_mode: full` there is no value in separating these two. it would increase complexity here without any gain. 
elastic is using them interchangeably in their documentation as well, so i would stick with using the same container for key and trust store for now. 

i will add a note about the format of the container tho (pkcs11 :wink: )